docs/modules/ROOT/pages/array-methods.adoc

```diff
@@ -0,0 +1,219 @@
+[[array-methods]]
+= Array methods
+
+Array methods allow us to modify existing property arrays in Update mutations within these entities:
+
+* `Node`
+* `Nested Nodes`
+* `Relationship properties`
+* `Interfaces`
+
+The following operators are available:
+
+* `_POP`
+* `_PUSH`
+
+The POP operator expects a single Int value as input
+
+The PUSH operator conforms to the type of input defined in the type definition.
+
+== Examples
+
+=== Array pop
+Suppose we have the following type definition, a Movie with a property array called tags:
+
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String
+    tags: [String]
+}
+----
+
+We can pop from this `tags` property array.
+
+Before: `['a', 'b', 'c']`
+
+After: `['a', 'b']`
+
+
+[source, graphql, indent=0]
+----
+mutation {
+    updateMovies (update: { tags_POP: 1 }) {
+        movies {
+            title
+            tags
+        }
+    }
+}
+----
+
+Or, for more than one property from the array:
+
+Before: `['a', 'b', 'c']`
+
+After: `['a']`
+
+[source, graphql, indent=0]
+----
+mutation {
+    updateMovies (update: { tags_POP: 2 }) {
+        movies {
+            title
+            tags
+        }
+    }
+}
+----
+
+Similarly, you can have multiple array property fields and update them in the same query:
+
+
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String
+    tags: [String]
+    moreTags: [String]
+}
+----
+
+We can push to both the `tags` and `moreTags` property arrays.
+
+
+Before: 
+```
+    tags: ['a', 'b', 'c']
+    moreTags: ['x', 'y', 'z']
+```
+
+After:
+```
+    tags: ['a', 'b']
+    moreTags: ['x']
+```
+
+[source, graphql, indent=0]
+----
+mutation {
+    updateMovies (update: { tags_POP: 1, moreTags_POP: 2 }) {
+        movies {
+            title
+            tags
+            moreTags
+        }
+    }
+}
+----
+
+=== Array push
+Suppose we have the following type definition, a Movie with a property array called tags:
+
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String
+    tags: [String]
+}
+----
+
+We can push to this `tags` property array.
+
+
+Before: `['some tag']`
+
+After: `['some tag', 'another tag']`
+
+[source, graphql, indent=0]
+----
+mutation {
+    updateMovies (update: { tags_PUSH: "another tag" }) {
```

Comment:
The documentation only gives an example of pushing a single value - it would be good to document pushing multiple values, i.e. an array to an array. Great documentation!